src/Tizen.Sensor/Tizen.Sensor/Sensor.cs

```diff
@@ -545,5 +638,623 @@ private void DestroyHandles()
+            }
+
+        }
+
+        /// <summary>
+        ///  Queries the recorded data asynchronously.
+        /// </summary>
+        /// <param name="type">
+        /// Sensor type.
+        /// </param>
+        /// <since_tizen> 6 </since_tizen>
+        public void RecorderRead(int type)
+        {
+            _sensorRecorderDataCallBack = (int _type, int data, int remains, int _error, Int64 _userData) =>
+            {
+                RecorderDataEventArgs e = new RecorderDataEventArgs(_type, data, remains, _error, _userData)
+                {
+                };
+
+                if (_error != 0)
+                {
+                    Log.Error(Globals.LogTag, "Error in Sensor Recorder Data aynchronously CallBack");
+                }
+
+                RecorderDataReceived?.Invoke(null, e);
+                Log.Debug(Globals.LogTag, "Recorder data callback recieved");
+                return true;
+            };
+
+            int error = Interop.SensorRecoder.RecorderRead(type, recorderQuery, _sensorRecorderDataCallBack, IntPtr.Zero);
+            if (error != (int)SensorError.None)
+            {
+                Log.Error(Globals.LogTag, "Error in asynchronously reading recorder");
+                throw SensorErrorFactory.CheckAndThrowException(error, "Sensor.RecorderRead Failed");
+            }
+
+        }
+
+        /// <summary>
+        ///  Starts to record a given sensor type.
+        /// </summary>
+        /// <param name="type">
+        /// A sensor type to be recorded.
+        /// </param>
+        /// <since_tizen> 6 </since_tizen>
+        public void RecorderStart(int type)
+        {
+            int error = Interop.SensorRecoder.RecorderStart(type, recorderOption);
+            if (error != (int)SensorError.None)
+            {
+                Log.Error(Globals.LogTag, "Error in starting recorder");
+                throw SensorErrorFactory.CheckAndThrowException(error, "Sensor.RecorderStart Failed");
+            }
+
+        }
+
+        /// <summary>
+        ///  Stops recording a given sensor type.
+        /// </summary>
+        /// <param name="type">
+        /// A sensor type to be recorded.
+        /// </param>
+        /// <since_tizen> 6 </since_tizen>
+        public void RecorderStop(int type)
+        {
+            int error = Interop.SensorRecoder.RecorderStop(type);
+            if (error != (int)SensorError.None)
+            {
+                Log.Error(Globals.LogTag, "Error in starting recorder");
+                throw SensorErrorFactory.CheckAndThrowException(error, "Sensor.RecorderStop Failed");
+            }
+
+        }
+
+        /// <summary>
+        ///  This function creates a sensor provider handle with a given URI.
+        /// </summary>
+        /// <param name="uri">
+        /// The URI of sensor to be created.
+        /// </param>
+        /// <since_tizen> 6 </since_tizen>
+        public void CreateProvider(string uri)
```

Comment:
Expect something a created object, but return type is void. 
What is the means of Create?